lib/capybara/driver/node.rb

```diff
@@ -138,6 +156,27 @@ def inspect
       def ==(other)
         eql?(other) || (other.respond_to?(:native) && native == other.native)
       end
+
+      protected
+
+      def normalize_spacing(text)
+        text
+          .delete(REMOVED_CHARACTERS)
+          .tr(SQUEEZED_SPACES, ' ')
+          .squeeze(' ')
+          .sub(LEADING_SPACES, '')
+          .sub(TRAILING_SPACES, '')
+          .tr(NON_BREAKING_SPACE, ' ')
```

Comment:
Then preserve NBSP

Reply:
None of these new methods should be in this class

Reply:
Will move to the implied new module mentioned above.